src/web/core/src/main/resources/GeoServerApplication_ca.properties

```diff
@@ -21,23 +21,23 @@ AbstractDataAccessPage.cantGetDataStoreFactory = No s'ha trobat la factoria del
 AbstractDataAccessPage.dataSrcName             = Nom de l'origen de dades
 AbstractDataAccessPage.workspace               = Espai de treball
 
-AbstractLayerGroupPage.bounds                  = L\u00edmits
-AbstractLayerGroupPage.chooseLayer             = Sel\u00b7lecciona una nova capa
-AbstractLayerGroupPage.chooseStyle             = Sel\u00b7lecciona un estil alternatiu
-AbstractLayerGroupPage.description             = Edita els continguts d'un grup de capes
-AbstractLayerGroupPage.down                    = Mou la capa a sota
-AbstractLayerGroupPage.duplicateGroupNameError = Ja existeix un grup de capes anomenat ${name}
-AbstractLayerGroupPage.notFound                = No s'ha trobat el grup de capes "{0}"
-AbstractLayerGroupPage.oneLayerMinimum         = El grup de capes ha de contenir al menys una capa
-AbstractLayerGroupPage.th.code                 = Codi\u0009
-AbstractLayerGroupPage.th.defaultStyle         = Estil per defecte
-AbstractLayerGroupPage.th.description          = Descripci\u00f3
-AbstractLayerGroupPage.th.layer                = Capa
-AbstractLayerGroupPage.th.position             = Posici\u00f3
-AbstractLayerGroupPage.th.remove               = Esborra
-AbstractLayerGroupPage.th.style                = Estil
-AbstractLayerGroupPage.title                   = Grup de capes
-AbstractLayerGroupPage.up                      = Mou la capa a sobre
+LayerGroupEditPage.bounds                  = L\u00edmits
```

Comment:
Everything looks good! 

The resource file src/web/core/src/main/resources/GeoServerApplication.properties is synchronized from github (master) into Transifex project https://www.transifex.com/GeoServer/geoserver (once merged back this pull into master). Every Translation team gets a message that the resource has been changed. 
I would suggest to add translated Cataln properties at Transifex (makes it easier to update from Transifex) and remove changes for this pull request for src/web/core/src/main/resources/GeoServerApplication_ca.properties

However, its just a suggestion.

At the moment of writing @oscarfonts managing Catalan Translation Team, @NielsCharlier feel free to join and help improving translations.